### PR TITLE
Add refresh step after pool create to make vol available

### PR DESCRIPTION
**Pull request by kylazhang**
Signed-off-by: Kyla Zhang <weizhan@redhat.com>

### CODE REVIEW

**Comment by chunfuwen** [comment]
is any result available?

**Comment by kylazhang** [comment]
```
# avocado run --vt-type libvirt virsh.blockcopy.positive_test.non_acl.block_disk.disk_t.default.volume_type.blockdev.no_shallow.finish_option
JOB ID     : b852d4176519cb9bae51f79f11bc67cb6ea46948
JOB LOG    : /root/avocado/job-results/job-2021-03-09T02.35-b852d41/job.log
 (1/1) type_specific.io-github-autotest-libvirt.virsh.blockcopy.positive_test.non_acl.block_disk.disk_t.default.volume_type.blockdev.no_shallow.finish_option: PASS (82.68 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 83.39 s
```

File: virttest/utils_test/libvirt.py [hunk 1]
```diff
@@ -1168,6 +1168,9 @@ def pre_pool(self, pool_name, pool_type, pool_target, emulated_image,
         xml_str = virsh.pool_dumpxml(pool_name)
         logging.debug("New prepared pool XML: %s", xml_str)
 
+        logging.info("refresh pool")
```
Comment (pevogam) [inline virttest/utils_test/libvirt.py:1171]: Can you capitalize this and write it as a whole sentence like
```python
logging.info("Refreshing pool")
```

**Review by pevogam — CHANGES_REQUESTED** [review]

File: virttest/utils_test/libvirt.py (hunk 1 above)
Comment (kylazhang) [reply virttest/utils_test/libvirt.py]: Fixed, thanks

**Review by pevogam — APPROVED** [review]

**Review by dzhengfy — APPROVED** [review]
LGTM

**Pull request merged**